templates/bridgeserver2.yaml

```diff
@@ -759,6 +759,83 @@ Resources:
       Period: 300
       Statistic: Average
       Threshold: 1
+  # Bucket for raw health data. Grant Synapse read-only permissions to this bucket. Only Bridge can
+  # write to thise bucket.
+  RawHealthDataBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
```

Comment:
is this a synapse external storage location?  if so your missing the CORS configuration that's specified in the [docs](https://docs.synapse.org/articles/custom_storage_location.html).

example is here: https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/s3/sc-s3-synapse-ra-v1.0.0.yaml#L50